PyInstaller/building/build_main.py

```diff
@@ -558,12 +558,9 @@ def _check_python_library(self, binaries):
             binaries.append((os.path.basename(python_lib), python_lib, 'BINARY'))
             logger.info('Using Python library %s', python_lib)
         else:
-            msg = """Python library not found! This usually happens on Debian/Ubuntu
-where you need to install Python library:
-
-  apt-get install python3-dev
-  apt-get install python-dev
```

Comment:
@matysek Thanks for reviewing! I reverted the error messages for Debian/Ubuntu.

Actually I didn't add message to recommend installing `libpythonX.Y` since it's a grandchild dependency of `python-dev` package and users don't have to install it explicitly.